smps.proto

```diff
@@ -143,6 +150,7 @@ message ServiceMeshConfig {
     string exp_group_uuid = 15;
     string exp_uuid = 16;
     string profile = 17;
+    string service_mesh = 18;
```

Comment:
To guarantee compatibility between SMPS implementations, we may want to use an enum for `service_mesh`.